src/ssl.c

```diff
@@ -16179,8 +16179,11 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
         return WOLFSSL_SUCCESS;
     }
 
-
+#if defined(WOLFSSL_QT)
```

Comment:
This should be switched based on OPENSSL_VERSION_NUMBER, as it appears that this is a OpenSSL version difference instead of specific to Qt.  WOLFSSL_QT sets OPENSSL_VERSION_NUMBER in wolfssl/openssl/opensslv.h.

Reply:
add #define OPENSSL_VESION_NUMBER 0x10100000L when defined(WOLFSSL_QT) case.  And guard the signature by it.